<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homeWindow</class>
 <widget class="QMainWindow" name="homeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="home_InfoLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter the text you want to analyse below:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="home_TextToAnalyse">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>681</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="home_AnalyseTextBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>480</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Analyse the text entered into the text box.</string>
    </property>
    <property name="text">
     <string>Analyse text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="home_OpenFileBtn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>530</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Open a text file to analyse its contents.</string>
    </property>
    <property name="text">
     <string>Open a file</string>
    </property>
   </widget>
   <widget class="QLabel" name="home_InfoLabel_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NOTE: each sentence must be on its own line.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
